features/matchers/have_enqueued_mail_matcher.feature

```diff
@@ -38,3 +38,40 @@ Feature: have_enqueued_mail matcher
       """
     When I run `rspec spec/mailers/user_mailer_spec.rb`
     Then the examples should all pass
+
+  Scenario: Checking mailer arguments
+    Given a file named "app/mailers/my_mailer.rb" with:
+      """ruby
+      class MyMailer < ApplicationMailer
+
+        def signup(user = nil)
+          @user = user
+
+          mail to: "to@example.org"
+        end
+      end
+      """
+    Given a file named "spec/mailers/my_mailer_spec.rb" with:
+      """ruby
+      require "rails_helper"
+
+      RSpec.describe MyMailer do
+        it "matches with enqueued mailer" do
+          ActiveJob::Base.queue_adapter = :test
+          # Works with plain args
+          expect {
+            MyMailer.signup('user').deliver_later
+          }.to have_enqueued_mail(MyMailer, :signup).with('user')
+          # Works with named parameters
+          expect {
+            MyMailer.with('foo' => 'bar').signup.deliver_later
+          }.to have_enqueued_mail(MyMailer, :signup).with('foo' => 'bar')
+          # Works also with both, named parameters match first argument
```

Comment:
WDYT of "Parameterize and pass an argument to the mailer"? And "Parameterize the mailer" for an example above.

Reply:
That's not bad.